docs/RELEASE_PROCESS.md

```diff
@@ -5,7 +5,8 @@
 - [ ] 3. Merge items in `PENDING.md` into the `CHANGELOG.md`. While doing this make sure that each entry contains links to issues/PRs for each item
 - [ ] 4. Summarize breaking API changes section under “Breaking Changes” section to the `CHANGELOG.md` to bring attention to any breaking API changes that affect RPC consumers.
 - [ ] 5. Tag the commit `{ .Release.Name }-rcN`
-- [ ] 6. Kick off 1 day of automated fuzz testing
-- [ ] 7. Release Lead assigns 2 people to perform [buddy testing script](/docs/RELEASE_TEST_SCRIPT.md) and update the relevant documentation
-- [ ] 8. If errors are found in either #6 or #7 go back to #2 (*NOTE*: be sure to increment the `rcN`)
-- [ ] 9. After #6 and #7 have successfully completed then merge the release PR and push the final release tag
+- [ ] 6. Open a branch & PR to merge the pending release back into `develop`. If any changes are made to the release branch, they must also be made to the pending develop merge, and both must pass tests.
+- [ ] 7. Kick off 1 day of automated fuzz testing
```

Comment:
Are we planning on doing this this time? We didn't last time, though if this is the GoS release, we probably should.

Reply:
I still don't think 1 *day* as opposed to 1 *hour* does much, but we can if we like.

Reply:
I totally disagree. Alot of the simulation bugs I found were only via using different configurations. Our simulation is also super slow due to having an exorbitant amount of governance slashing, so we don't simulate blocks super fast. (My unmerged pr to switch governance to use future ops helps alleviate this, but not fully)

Also its important to test many different configurations (e.g. different seeds). Even once we fix the aforementioned governance bugs, I think we should have a full days worth of testing. (Block size 100k or more nominally once we reduce governance proposal frequency) I suspect there are a large class of bugs that may only be exploitable when theres a large state tree built up.

Reply:
OK, let's try it!